src/chunkserver/register.cpp

```diff
@@ -63,12 +63,14 @@ int Register::RegisterToMDS(ChunkServerMetadata *metadata) {
         req.set_externalip(ops_.chunkserverExternalIp);
     }
     req.set_port(ops_.chunkserverPort);
+    req.set_blocksize(ops_.blockSize);
```

Comment:
Is this for verification on the mds side?

Reply:
yep